services/src/main/java/org/exoplatform/addons/gamification/storage/RealizationsStorage.java

```diff
@@ -19,7 +19,16 @@ public RealizationsStorage(GamificationHistoryDAO gamificationHistoryDAO) {
   public List<GamificationActionsHistoryDTO> getAllRealizationsByFilter(RealizationsFilter realizationFilter,
                                                                         int offset,
                                                                         int limit) {
-    List<GamificationActionsHistory> gamificationActionsHistoryList = gamificationHistoryDAO.findRealizationsByFilter(realizationFilter,
+    List<GamificationActionsHistory> gamificationActionsHistoryList = gamificationHistoryDAO.findAllRealizationsByFilter(realizationFilter,
+                                                                                                                      offset,
```

Comment:
bad formatting, please check the other blocks as well

Reply:
done !